src/main.rs

```diff
@@ -8,16 +8,18 @@
 #![test_runner(crate::kernel_tests::runner)]
 #![reexport_test_harness_main = "ktests_launch"]
 
+#[macro_use]
+extern crate static_assertions;
```

Comment:
Do we still need these two lines in Rust 2018/2021 edition? I was under the impression they could all be replaced with regular `use static_assertions::<name of the macro>` now

Reply:
I always forgot about this, fixed